requirements.txt

```diff
@@ -8,3 +8,5 @@ protobuf
 six
 petname
 google.cloud.logging
+matplotlib
```

Comment:
these are some heavy dependencies; matplotlib in particular will probably give folks headaches.  Could we maybe make this 'requirements-analysis.txt' so folks don't necessarily need to add them all to run the oneoffs

Reply:
Done